### PR TITLE
feat: remove ANONYMOUS field

### LINKED ISSUE #1346: Remove `ANONYMOUS` static field from the `Authentication` interface
`ANONYMOUS` is used only in tests, so it should be removed from the main code base

### PR DESCRIPTION
**Pull request by dgarus**
Closes #1346

### CODE REVIEW

File: rpm-adapter/src/test/java/com/artipie/rpm/http/RpmSliceS3ITCase.java [hunk 1]
```diff
@@ -39,16 +39,16 @@
 import org.testcontainers.Testcontainers;
 import org.testcontainers.containers.Container;
 import org.testcontainers.containers.GenericContainer;
+
```
Comment (ChGen) [inline rpm-adapter/src/test/java/com/artipie/rpm/http/RpmSliceS3ITCase.java:42]: Redundant empty line.

File: rpm-adapter/src/test/java/com/artipie/rpm/http/RpmSliceITCase.java [hunk 2]
```diff
@@ -35,12 +33,14 @@
 import org.testcontainers.containers.Container;
 import org.testcontainers.containers.GenericContainer;
```
Comment (ChGen) [inline rpm-adapter/src/test/java/com/artipie/rpm/http/RpmSliceITCase.java:35]: Redundant empty line.

File: rpm-adapter/src/test/java/com/artipie/rpm/http/RpmSliceDownloadITCase.java [hunk 3]
```diff
@@ -22,6 +23,8 @@
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.Optional;
+
```
Comment (ChGen) [inline rpm-adapter/src/test/java/com/artipie/rpm/http/RpmSliceDownloadITCase.java:27]: Redundant empty line.

File: pypi-adapter/src/test/java/com/artipie/pypi/http/PySliceTest.java [hunk 4]
```diff
@@ -28,11 +28,13 @@
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvSource;
```
Comment (ChGen) [inline pypi-adapter/src/test/java/com/artipie/pypi/http/PySliceTest.java:30]: Redundant empty line.

File: nuget-adapter/src/test/java/com/artipie/nuget/http/index/NuGetServiceIndexTest.java [hunk 5]
```diff
@@ -34,13 +27,20 @@
 import wtf.g4s8.hamcrest.json.JsonHas;
 import wtf.g4s8.hamcrest.json.JsonValueIs;
```
Comment (ChGen) [inline nuget-adapter/src/test/java/com/artipie/nuget/http/index/NuGetServiceIndexTest.java:29]: Redundant empty line.

File: helm-adapter/src/test/java/com/artipie/helm/http/HelmDeleteIT.java [hunk 6]
```diff
@@ -14,21 +14,23 @@
...
-import java.util.Optional;
-import java.util.Queue;
-import java.util.concurrent.ConcurrentLinkedQueue;
-import java.util.stream.Stream;
 import org.hamcrest.MatcherAssert;
 import org.hamcrest.core.IsEqual;
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
```
Comment (ChGen) [inline helm-adapter/src/test/java/com/artipie/helm/http/HelmDeleteIT.java:26]: Redundant empty line.

File: helm-adapter/src/test/java/com/artipie/helm/HelmSliceIT.java [hunk 7]
```diff
@@ -44,13 +33,23 @@
 import org.testcontainers.containers.Container;
 import org.testcontainers.containers.GenericContainer;
```
Comment (ChGen) [inline helm-adapter/src/test/java/com/artipie/helm/HelmSliceIT.java:35]: Redundant empty line.

File: go-adapter/src/test/java/com/artipie/http/GoSliceTest.java [hunk 8]
```diff
@@ -19,16 +20,18 @@
...
 import com.artipie.security.policy.PolicyByUsername;
-import java.util.concurrent.ExecutionException;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
 import org.apache.commons.lang3.tuple.ImmutablePair;
 import org.apache.commons.lang3.tuple.Pair;
 import org.hamcrest.MatcherAssert;
 import org.hamcrest.core.AllOf;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.ValueSource;
```
Comment (ChGen) [inline go-adapter/src/test/java/com/artipie/http/GoSliceTest.java:29]: Redundant empty line.

File: go-adapter/src/test/java/com/artipie/http/GoSliceITCase.java [hunk 9]
```diff
@@ -29,6 +30,8 @@
 import org.testcontainers.Testcontainers;
 import org.testcontainers.containers.GenericContainer;
```
Comment (ChGen) [inline go-adapter/src/test/java/com/artipie/http/GoSliceITCase.java:32]: Redundant empty line.

File: artipie-core/src/test/java/com/artipie/http/auth/AuthLoaderTest.java [hunk 10]
```diff
@@ -7,6 +7,9 @@
 import com.amihaiemil.eoyaml.Yaml;
 import com.artipie.ArtipieException;
 import java.util.Collections;
+
```
Comment (ChGen) [inline artipie-core/src/test/java/com/artipie/http/auth/AuthLoaderTest.java:10]: Redundant empty line.

**Review by ChGen — CHANGES_REQUESTED** [review]
@dgarus , there's an extra empty line inside imports block in every changed file.

**Comment by dgarus** [comment]
> @dgarus , there's an extra empty line inside imports block in every changed file.

This line is between java lib's imports and others. 
This is a standard rule in the Java world.

**Review by ChGen — DISMISSED** [review]
@dgarus , thanks.

**Pull request merged**